### PR TITLE
Add file path -> crc32 algorithm

### PR DESCRIPTION
**Pull request by MrCreosote**
Can use the base64 encoded crc to validate uploads like so:

```
In [56]: fields
Out[56]:
{'key': 'somefile',
 'AWSAccessKeyId': 'admin',
 'policy':
'eyJleHBpcmF0aW9uIjogIjIwMjQtMDgtMjZUMjM6MzE6MDBaIiwgImNvbmRpdGlvbnMiOiBbeyJidWNrZXQiOiAidGVzdCJ9LCB7ImtleSI6ICJzb21lZmlsZSJ9XX0=',
 'signature': 'Ys4tVhnbR1fUrMgpm+1DRxXsWnY=',
 'x-amz-checksum-crc32': b'bwUq9A=='}

In [57]: with open("Dockerfile") as f:
    ...:     files = {"file": ("somefile", f)}
    ...:     res = requests.post(url["url"], data=fields, files=files)

```

### CODE REVIEW

File: cdmtaskservice/s3/remote.py [hunk 1]
```diff
@@ -40,16 +41,31 @@ def calculate_etag(infile: Path, partsize: int) -> str:
...
+
+
+def crc32(infile: Path) -> bytes:
+    """Compute the CRC-32 checksum of the contents of the given file"""
+    # adapted from https://stackoverflow.com/a/59974585/643675
+    # Not really a good way to test the expanduser calls
+    _check_file(infile)
+    with open(infile.expanduser(), "rb") as f:
+        checksum = 0
+        while chunk := f.read(65536):
```
Comment (Tianhao-Gu) [inline cdmtaskservice/s3/remote.py:65]: can we replace `65536` to something like
`CHUNK_SIZE_64KB = 2**16`

Comment (MrCreosote) [reply cdmtaskservice/s3/remote.py]: Done

Comment (Tianhao-Gu) [reply cdmtaskservice/s3/remote.py]: 👍

**Review by Tianhao-Gu — APPROVED** [review]
👍

**Pull request merged**